### PR TITLE
Drupalize.me

### LINKED ISSUE #40: github webhooks seem to be broken
SyntaxError: Unexpected token u
    at Object.parse (native)
    at /home/willwh/lullabot/lubot/scripts/github.js:9:26
    at callbacks (/home/willwh/lullabot/lubot/node_modules/express/lib/router/index.js:164:37)
    at param (/home/willwh/lullabot/lubot/node_modules/express/lib/router/index.js:138:11)
    at pass (/home/willwh/lullabot/lubot/node_modules/express/lib/router/index.js:145:5)
    at Router._dispatch (/home/willwh/lullabot/lubot/node_modules/express/lib/router/index.js:173:5)
    at Object.router (/home/willwh/lullabot/lubot/node_modules/express/lib/router/index.js:33:10)
    at next (/home/willwh/lullabot/lubot/node_modules/express/node_modules/connect/lib/proto.js:193:15)
    at Object.urlencoded [as handle](/home/willwh/lullabot/lubot/node_modules/express/node_modules/connect/lib/middleware/urlencoded.js:38:27)
    at next (/home/willwh/lullabot/lubot/node_modules/express/node_modules/connect/lib/proto.js:193:15)

### PR DESCRIPTION
**Pull request by willwh**
resolves #40, #41, #42, #44
selfie-0

### CODE REVIEW

**Comment by willwh** [comment]
Commit messages a little dirty :+1:

File: scripts/hackpad.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    botText = bot.helpers.utils.startsBot(text);
+    // If the bot is addressed, continue.
+    if (botText !== false) {
+      text = botText;
+
+      // Help summary.
+      if (text === 'make me a hackpad') {
+        var uuid1 = uuid.v4();
+        var hackurl = 'https://lullabot.hackpad.com/' + uuid1;
```
Comment (justafish) [inline scripts/hackpad.js]: I think this should be a configurable URL?

Comment (eojthebrave) [inline scripts/hackpad.js]: Maybe for now we should add a `LUBOT_HACKPAD_URL` env variable like we do with all the other settings. We can come back and make this better once we figure out how to handle config in #16.

Comment (willwh) [inline scripts/hackpad.js]: Proposing a LUBOT_HACKPAD_DOMAIN so we are just using sub.domain.com or domain.com :)

Comment (eojthebrave) [inline scripts/hackpad.js]: That works.

**Comment by m4olivei** [comment]
@willwh is this branch the future of lubot?  I'd love to try testing it out.  How do you get it running locally now?  When I try running it I get:

```
$ node lubot.js

fs.js:432
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: ENOENT, no such file or directory '/etc/ssl/box.key'
    at Object.fs.openSync (fs.js:432:18)
    at Object.fs.readFileSync (fs.js:289:15)
    at Object.<anonymous> (/Users/matt/code/lubot/lubot.js:21:13)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
```

Looks like it's because I have nothing in `/etc/ssl`.  I'm guessing I need to add a CA, CERT and KEY file there.  Also guessing that I do that with `openssl` utility (I'm a Mac user if it matters)?  I tried, but failed to get that going.  How do I do that?

**Pull request closed without merging**